arrow-inject-compiler-plugin/src/test/kotlin/arrow/inject/compiler/plugin/runners/BaseTestRunner.kt

```diff
@@ -46,7 +47,11 @@ fun TestConfigurationBuilder.commonFirWithPluginFrontendConfiguration() {
     dependencyKind = DependencyKind.Binary
   }
 
-  useConfigurators(::PluginAnnotationsProvider, ::ExtensionRegistrarConfigurator)
+  useConfigurators(
+    ::PluginAnnotationsConfigurator,
+    ::ExtensionRegistrarConfigurator,
+    ::LinkViaSignaturesConfigurator,
```

Comment:
Would be good to try if we still need `LinkViaSignaturesConfigurator` for tests to pass or if the way we encoded it in the pair session today already takes care of it.